infra/terraform/variable.tf

```diff
@@ -28,38 +28,6 @@ variable "fixed_ip" {
   description = "Desirated IP address for instance"
 }
 
-variable "branch" {
-  type        = string
-  default     = "main"
-  description = "Branch name to deploy"
-}
```

Comment:
Keep branch in tf variable because the deploy.sh script need this

Reply:
I don't understand, I thought the variables for deploy.sh should go in the env.json file ?

Reply:
true I fix this